includes/multi-currency/Currency.php

```diff
@@ -66,24 +66,32 @@ class Currency implements \JsonSerializable {
 	 */
 	private $last_updated;
 
+	/**
+	 * Instance of WC_Payments_Localization_Service.
+	 *
+	 * @var WC_Payments_Localization_Service
+	 */
+	private $localization_service;
+
 	/**
 	 * Constructor.
 	 *
-	 * @param string   $code Three letter currency code.
-	 * @param float    $rate The conversion rate.
-	 * @param int|null $last_updated The time this currency was last updated.
+	 * @param WC_Payments_Localization_Service $localization_service Three letter currency code.
```

Comment:
Fixing the argument description.

```suggestion
	 * @param WC_Payments_Localization_Service $localization_service Localization service instance.
```

Reply:
Thank you for catching this, I've applied your suggested changes 💯